src/components/DisplayTaxonName.js

```diff
@@ -1,159 +1,57 @@
 // @flow
-
-import { Text } from "components/styledComponents";
-import _ from "lodash";
+import classnames from "classnames";
+import { Body1, Body3 } from "components/SharedComponents";
+import { View } from "components/styledComponents";
 import type { Node } from "react";
 import React from "react";
+import { useTranslation } from "react-i18next";
+import { generateTaxonPieces } from "sharedHelpers/taxon";
 
 type Props = {
   item: Object,
+  layout?: string,
```

Comment:
If I'm understanding correctly, this can have the value of `list` when the name should be displayed as a single line as it is in the observation list... but this component will be used in a lot of places, not just in list vs grid scenarios, so maybe what we need here is `layout?: "horizontal" | "vertical"`